<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>DownloadDialog</class>
    <widget class="QDialog" name="DownloadDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>342</width>
                <height>141</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Dialog</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
            <item row="2" column="0">
                <widget class="QLabel" name="password_label">
                    <property name="text">
                        <string>Password</string>
                    </property>
                </widget>
            </item>
            <item row="0" column="0" colspan="2">
                <widget class="QComboBox" name="source_combobox"/>
            </item>
            <item row="1" column="1">
                <widget class="QLineEdit" name="username_lineedit">
                    <property name="text">
                        <string/>
                    </property>
                </widget>
            </item>
            <item row="2" column="1">
                <widget class="QLineEdit" name="password_lineedit">
                    <property name="inputMethodHints">
                        <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
                    </property>
                    <property name="echoMode">
                        <enum>QLineEdit::Password</enum>
                    </property>
                </widget>
            </item>
            <item row="1" column="0">
                <widget class="QLabel" name="username_label">
                    <property name="text">
                        <string>Username</string>
                    </property>
                </widget>
            </item>
            <item row="3" column="0" colspan="2">
                <widget class="QDialogButtonBox" name="buttonBox">
                    <property name="standardButtons">
                        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections>
        <connection>
            <sender>buttonBox</sender>
            <signal>rejected()</signal>
            <receiver>DownloadDialog</receiver>
            <slot>reject()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>170</x>
                    <y>113</y>
                </hint>
                <hint type="destinationlabel">
                    <x>170</x>
                    <y>70</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>
